src/Illuminate/Support/Str.php

```diff
@@ -886,4 +887,18 @@ public static function createUuidsNormally()
     {
         static::$uuidFactory = null;
     }
+
+    /**
+     * Generate a ULID.
+     *
+     * @param  bool  $lowercase
+     * @param  string|null  $dateTime
+     * @return \Ulid\Ulid
+     */
+    public static function ulid(bool $lowercase = false, string $dateTime = null)
```

Comment:
Maybe will be better using DateTimeInterface instead string?

Reply:
i changed that from string to DateTimeInterface